elastic/integrations
# Jolokia Input

- Enhancement

## What does this PR do?

This PR creates the Jolokia Input Package

## Checklist


## How to test this PR locally

Bring up the Jolokia Input in Kibana, run the Jolokia agent and check for metrics being received on Kibana.

- Closes https://github.com/elastic/integrations/issues/4498

## Screenshots
<img width="1239" alt="Screenshot 2022-11-03 at 11 02 09 AM" src="https://user-images.githubusercontent.com/102962586/199656224-910f5562-c4bd-4d11-8fb5-237f73efe791.png">
<img width="1693" alt="Screenshot 2022-11-03 at 11 14 17 AM" src="https://user-images.githubusercontent.com/102962586/199656234-0dea539a-3fa4-4491-8baa-aab44c788fe6.png">
